src/components/TodoList/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoList';
```

Comment:
In my personal opinion using export * for just one file in a folder is an overkill.